### PR TITLE
Feature/create'books

### PR DESCRIPTION
**Pull request by MacCrazyman**
# Add and remove books 📚 

This branch include features to add and remove books:
* User Redux to check the state of the books array.
* Update UI according to state.
* Use `useDispatch( )` to add or remove books from store.
* Implement ducks pattern. 🦆

### CODE REVIEW

File: src/components/books.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      author: 'Leonardo Pareja',
+    };
+    dispatch(addBook(newBook));
+    e.target.bookTitle.value = '';
+    e.target.categories.value = '';
+  };
+
+  // const deleteBook = (id) => {
+  //   useDispatch(removeBook(id));
+  // };
```
Comment (bimbolabuari) [inline src/components/books.jsx]: Let's remove this commented code in order to make your code look cleaner ✊

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @MacCrazyman, 

You have done a good job so far 🚀  

## Highlights 🎉 

- Import createStore, combineReducers  and applyMiddleware  from redux.
- Import booksReducer from `src/redux/books`.
- Usage of combineReducers to create a reducer.
- `booksReducer` is added to combineReducers.
- Usage of array.filter method to remove book.

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: src/components/books.jsx (hunk 1 above)
Comment (MacCrazyman) [reply src/components/books.jsx]: Thanks for the suggestion.

**Review by ZeenatLawal — APPROVED** [review]
Hi @MacCrazyman, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**